internal/exec/engine.go

```diff
@@ -72,7 +78,23 @@ func (e Engine) Run(stageName string) error {
 	e.logReport(r)
 	if err != nil && err != providers.ErrNoProvider {
 		e.Logger.Crit("failed to acquire system base config: %v", err)
-		return err
+	}
+
+	baseConfigNotFound := false
+	if err == providers.ErrNoProvider {
+		baseConfigNotFound = true
+	}
+
+	configExists, err := executil.PathExists(e.ConfigCache)
+	if err != nil {
+		e.Logger.Info("failed to read cached config: %v", err)
+	}
+	// TODO(sohan): This is a hack to emit the journald message only
+	// once for the base config (https://github.com/coreos/ignition/issues/950)
+	if !configExists && baseConfigNotFound {
```

Comment:
`!baseConfigNotFound`... which instead of the double-negation, WDYT about flipping the variable semantic to be `baseConfigFound` so it's easier to grok?